ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -2,3 +2,21 @@
 # defaults file for installer
 cache_enabled: false
 provision_cache_store: "{{ ansible_env.HOME }}/rhcos_image_cache/"
+
+# The containers registry port
```

Comment:
asking for opinion - should we include comments for these vars considering they are captured in the inventory/hosts.sample?

Side note: can we remove the empty line?

Reply:
I guess it is a matter of preference. A short description in the file makes its a little easier when modifying the playbook. But it does clutter the file a little more.
We can remove the comments if you wish and the space as well

Reply:
changed will be in next commit